src/clj/harja/kyselyt/tieturvallisuusverkko.sql

```diff
@@ -19,3 +19,12 @@ FROM (SELECT ST_Union(sisempi.geom) u
               AND ST_Intersects(:saatugeometria::GEOMETRY, geometria::geometry)
            ) AS sisempi
      ) AS ulompi;
+
+-- name: hae-urakan-tieturvallisuusverkko-kartalle
+SELECT
+    ST_INTERSECTION(geometria, u.alue) AS sijainti
```

Comment:
Ihan vain kommenttina heitän, että olisin itse ensimmäisenä ajatellut käyttää ST_Dwithin funkkaria, eli ikään kuin kaikki, jotka on urakan sisällä (hoitourakat on tämmösiaä alueita. ei toimisi valaistusurakoilla, mutta ei ole ehkä tarvettakaan) 
Jos nousee toinen asia. Pitäiskö rajoittaa urakkatyypillä tätä hakua, ettei valaistus tai kanavaurakoille koiteta näitä hakea?

Reply:
ST_Dwithin toimisi myös. Ajattelin aiemmin ST_Withiniä, joka ei toimisi (tieosatt, jotka on urakan rajan molemmilla puolin jäisi pois). Kokeilen onko ST_Dwithin yhtään tehokkaampi.

Urakkatyypillä rajoitus on hyvä idea, lisään sen tähän.

Reply:
ST_Dwithin on noin 10 kertaa hitaampi kuin ST_INTERSECTS, pidetään se. Urakan tyypin tarkastus lisätty